asapdiscovery/data/pdb.py

```diff
@@ -30,39 +29,51 @@ def download_pdb_structure(pdb_id: str, directory: str):
         The PDB ID of interest.
     directory: str or Path, default=user_cache_dir
         The directory for saving the downloaded structure.
+    file_format: str, optional
+        Indicates whether you would like to download the entry in pdb ("pdb") or cif format ("cif"),
+        or the first biological assembly in cif format ("cif1"). Defaults to "pdb".
 
     Returns
     -------
-    : Path or False
-        The path to the the downloaded file if successful, else False.
+    file_path : Path or False
+        The path to the downloaded file if successful, else False.
     """
     from .utils import download_file
     import os
 
-    # check for structure in PDB format
-    url = f"https://files.rcsb.org/download/{pdb_id}.pdb"
-    pdb_path = os.path.join(directory, f"rcsb_{pdb_id.upper()}.pdb")
-    if not os.path.exists(pdb_path):
-        print("Downloading PDB entry in PDB format ...")
-        if download_file(url, pdb_path):
-            return pdb_path
-    else:
-        return pdb_path
-
-    # check for structure in CIF format
-    url = f"https://files.rcsb.org/download/{pdb_id}.cif"
-    cif_path = os.path.join(directory, f"rcsb_{pdb_id.upper()}.cif")
-    if not os.path.exists(cif_path):
-        print("Downloading PDB entry in CIF format ...")
-        if download_file(url, cif_path):
-            return cif_path
+    url_base_str = f"https://files.rcsb.org/download/"  # base str to use for URLs
+    # Dictionary with allowed formats and their upstream basenames
+    format_to_basename = {
+        "pdb": f"{pdb_id.lower()}.pdb",
+        "cif": f"{pdb_id}.cif",
+        "cif1": f"{pdb_id.lower()}-assembly1.cif"
```

Comment:
Actually it doesn't matter
```suggestion
        "pdb": f"{pdb_id}.pdb",
        "cif": f"{pdb_id}.cif",
        "cif1": f"{pdb_id}-assembly1.cif"
```

Reply:
@kaminow

Reply:
I'm actually not sure about this, I just maintained the same structure as before, but do notice that the file names are case-sensitive. Are we sure the case doesn't matter here and this works as expected?

Reply:
They end up being downloaded as the same thing, but the pdb id case in the filename in RCSB might not be the same for every format.

Reply:
You can use any case for downloading any of the formats!

Reply:
even camel case =P

Reply:
I think the main thing is to make sure that the downloaded files have the same name regardless of what case the query was

Reply:
Ohhhhhhh sorry I got it. I missed that the local path now uses the `basename`. So maybe changing to upper case here is good.